src/main/scala/org/broadinstitute/dsde/workbench/sam/api/ManagedGroupRoutes.scala

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.dsde.workbench.sam.api
+
+// package object for implicit ErrorReportSource
+import org.broadinstitute.dsde.workbench.model.UserInfo
+import org.broadinstitute.dsde.workbench.sam._
+
+// IntelliJ may highlight these imports as unused, but they're needed for the json formatting to work properly
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import org.broadinstitute.dsde.workbench.sam.model._
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server
+import akka.http.scaladsl.server.Directives._
+import org.broadinstitute.dsde.workbench.model.{ErrorReport, WorkbenchExceptionWithErrorReport}
+import org.broadinstitute.dsde.workbench.model.WorkbenchIdentityJsonSupport._
+import org.broadinstitute.dsde.workbench.sam.model.ResourceId
+import org.broadinstitute.dsde.workbench.sam.service.ManagedGroupService
+
+import scala.concurrent.ExecutionContext
+
+/**
+  * Created by gpolumbo on 2/20/2018.
+  */
+trait ManagedGroupRoutes extends UserInfoDirectives with SecurityDirectives {
+  implicit val executionContext: ExecutionContext
+
+  val managedGroupService: ManagedGroupService
+
+  def groupRoutes: server.Route = requireUserInfo { userInfo =>
+    path("group" / Segment) { groupId =>
+      get {
+        handleGetGroup(groupId)
+      } ~
+      post {
+        handlePostGroup(groupId, userInfo)
+      } ~
+      delete {
+        handleDeleteGroup(groupId, userInfo)
+      }
+    }
+  }
+
+  private def handleGetGroup(groupId: String) = {
+    complete (
```

Comment:
@breilly2 excellent question.  @dvoet and I talked about this a bit and my understanding was that the decision for now was to leave it as is since we aren't really revealing any information and somebody could "guess" if a group exists simply by trying to POST groups with names they're looking for.